src/ops/fuelup_toolchain/uninstall.rs

```diff
@@ -20,13 +20,17 @@ pub fn uninstall(command: UninstallCommand) -> Result<()> {
     };
 
     if !toolchain.exists() {
-        info!("toolchain '{}' does not exist", &toolchain.name);
+        println_warn(format!("Toolchain '{}' does not exist", &toolchain.name));
         return Ok(());
     }
 
+    if config.list_toolchains()?.len() == 1 {
+        bail!("Cannot uninstall the last toolchain");
+    }
+
     match toolchain.uninstall_self() {
         Ok(_) => {
-            info!("toolchain '{}' uninstalled", &toolchain.name);
+            println!("Toolchain '{}' uninstalled", &toolchain.name);
```

Comment:
```suggestion
            info!("Toolchain '{}' uninstalled", &toolchain.name);
```

We shouldn't always use tracing instead of printlns so that all output is captured in the log files.